src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -46,7 +46,6 @@ import { IndexPatternMissingIndices } from '../lib';
 import { findByTitle } from '../utils';
 import { DuplicateIndexPatternError } from '../errors';
 
-const indexPatternCache = createIndexPatternCache();
```

Comment:
@mattkime @ppisljar it looks like an issue while unifying client/server code.

This approach unacceptable for server side code. `indexPatternCache` variable should not declared as instance variable.

Reply:
also I think it should be backported into `7.10`, `7.11` as a security issue.